uvcombine/uvcombine.py

```diff
@@ -256,8 +303,24 @@ def smoothing(combo, targres):
        Combined image
     targres : float
        The HPBW of the smoothed image (in units of arcsecond)
-    """
+    origfwhm : float
+       The original HPBW of input image (in units of arcsecond)
+    pixscale : float
+       The pixscale of the input image (in units of arcsecond)
 
+    Returns
+    -------
+    combo : float array
+       Smoothed image
+    """
+    fwhm = np.sqrt(8*np.log(2))
+    kernel_size=((targres/fwhm)**2-(hiresfwhm/fwhm)**2)**0.5
+    pixel_n = kernel_size/pixscale
+    
+    #smooth the image using gaussian 2d kernel
+    gauss_kernel =Gaussian2DKernel(pixel_n)
```

Comment:
Just a general note - in astropy-affiliated packages, we try to hold to PEP8 conventions, which means keeping a space on either side of `=` signs.